### DISCUSSION TITLE
Align mode name and command/action name

### DISCUSSION THREAD

**Discussion by turhobr** [davatorium/rofi#1946] [Theming] [answered]
hi, i'm using glue_pro_blue theme and i wanna ask if it's possible to align mode name and name of command/action because as you can see here: https://ctrlv.link/ViXF name after "window" mode has much bigger padding after "window" than after "drun"
![image](https://github.com/davatorium/rofi/assets/28293642/d0a3b455-fbf0-487b-ba8d-113eb51455c0)

**Comment by DaveDavenport** [comment, marked as answer]
it looks like your workspace names are empty. 
Try removing them from -window-format 

```bash
rofi -window-format  '{c} {t}' -show combi 
```

to align after the mode name, use a tab?

```bash
rofi -window-format  '{c} {t}' -show combi -combi-display-format '{mode}	{text}'
```
